apps/i18n/common/en_us.json

```diff
@@ -1866,6 +1866,29 @@
   "myProjects": "My Projects",
   "myProfessionalLearningCourses": "My Professional Learning Courses",
   "myWorkshops": "My Workshops",
+
+  "accountInformationAccountInformation": "Account information",
+  "accountInformationVerifiedTeacher": "Verified Teacher",
+  "accountInformationDisplayName": "Display Name",
+  "accountInformationUsername": "Username",
+  "accountInformationDisplayNameHintTeacher": "This is what your students see.",
+  "accountInformationEmail": "Email address",
+  "accountInformationEmailHint": "Make sure you can receive emails to this email address.",
+  "accountInformationPassword": "Password",
+  "accountInformationPasswordConfirmation": "Password confirmation",
+  "accountInformationCurrentPassword": "Confirm current password",
+  "accountInformationCurrentPasswordHint": "We need your current password to confirm changes.",
+  "accountInformationUpdateAccountInformation": "Update account information",
+  "accountInformationNoPasswordBecauseSponsored": "See your teacher to manage your password.",
+  "accountInformationGender": "Gender (optional)",
+  "accountInformationState": "State",
+  "accountInformationUpdateFieldParentPermissionRequired": "(You must obtain parental permission before updating this field)",
+  "accountInformationUpdateEmail": "Update email",
+  "accountInformationSelectState": "Select state",
+  "accountInformationReviewErrors": "Review errors above and try again.",
+  "accountInformationUpdateSuccess": "Account information successfully updated.",
+  "accountInformationEmailUpdateSuccess": "Email successfully updated.",
+
```

Comment:
This comment is more so for future changes since this PR has already been merged. 
We prefer adding underscores to separate between component name and string name, for example in this case, it would be more readable to have used: `"accountInformation_accountInformation"`, `"accountInformation_verifiedTeacher"`, `"accountInformation_displayName"`, etc.